src/Components/CommonComponents/ListCardComponent.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import { Box, Container } from '@mui/material';
+
+const ListCardComponent = () => {
+  const [listData, setListData] = useState();
+
+  useEffect(() => {
+    //method for fetch data and update state
+    //setListData(data);
```

Comment:
please delete this comment. The code must be enougth documentation to itself. Also, because its a reusable component it will not use service/redux because will receive all information from props